meta-xt-control-domain/recipes-guest/doma/u-boot-android-bazel.bb

```diff
@@ -0,0 +1,74 @@
+inherit base
+
+# PARAMETERS
```

Comment:
You can hardly call these variables parameters if you didn't leave means to override them. Why not use `?=` syntax?

Reply:
Hi @lorc, done.